app/assets/stylesheets/header.scss

```diff
@@ -162,45 +163,43 @@ body > header {
         overflow: hidden;
         position: relative;
         max-height: 325px;
-      }
 
-      .notification_element {
-        border-bottom: 1px solid $border-grey;
-        padding: 5px;
-        min-height: 35px;
-        line-height: 18px;
-
-        > .avatar {
-          height: 35px;
-          width: 35px;
-          float: left;
-        }
+        .media.stream_element {
+          border-bottom: 1px solid $border-grey;
+          padding: 5px;
+          min-height: 35px;
+          line-height: 18px;
+          margin: 0;
 
-        .notification_message {
-          margin-left: 40px;
-          .tooltip { position: fixed; }
-        }
+          img.avatar {
+            width: 35px;
+            height: 35px;
+          }
+
+          .pull-right{
+            margin-top: 20px;
+            .aspect_membership_dropdown{ display: none; }
+          }
 
-        .unread-toggle {
-          padding: 9px 9px;
-          position: relative;
           .entypo {
             cursor: pointer;
             color: lighten($black,75%);
             font-size: 17px;
             line-height: 17px;
+            padding: 10px;
           }
-          .tooltip {
-            position: fixed;
-          }
```

Comment:
You removed this and only added it for unread notifications. We need the same for read notifications so please revert this.